### ISSUE TITLE
Global Styles not being applied correctly on front end

### ISSUE THREAD

**Issue by p-jackson** [Automattic/themes#3588]
## Steps to replicate

- Install and activate Seedlet Blocks on a standalone site, install GB
- Open site editor and change the text colour using Global Styles and save the change
- Notice that the text in the editor changes
- Refresh the page, notice the text colour change has been saved ![editor](https://user-images.githubusercontent.com/1500769/113075821-36c39080-922a-11eb-9ea4-0ad3b24b5268.jpg)

- Load front end

## Result

The text still appears black on the front end

![frontend](https://user-images.githubusercontent.com/1500769/113075841-42af5280-922a-11eb-97a0-5cf7a1ef2f27.jpg)

## Expected

Changes to global styles should be reflected on the front-end of the site

Discovered while debugging a Global Styles issue (Automattic/wp-calypso#50953). This issue was happening with the Seedlet Blocks theme, but not TT1 Blocks.

I found this code that was overriding the colours on the front end:
<img width="816" alt="Screenshot 2021-03-29 at 11 59 17 AM" src="https://user-images.githubusercontent.com/1500769/113076013-a89bda00-922a-11eb-9286-ab2e04b105e0.png">

Maybe it's something that was needed in earlier iterations of Global Styles?

**Comment by pbking** [comment]
This doesn't seem to be happening on the latest build of Seedlet-blocks.  Can you re-evaluate?

**Comment by p-jackson** [comment]
Is there somewhere I can download the latest "blessed" builds? When I build them myself from `trunk` I'm getting errors so can't verify.

**Comment by pbking** [comment]
There shouldn't be a need to build any of the themes on the trunk branch unless you make changes.

I'm also not experiencing any issues building any of the themes.  I am happy to help you with the build if you're having issues though.

The version of seedlet-blocks that's installed on wp.com for testing should be fine to validate that the issue is no longer occurring.

**Comment by p-jackson** [comment]
> The version of seedlet-blocks that's installed on wp.com for testing should be fine

Unfortunately Automattic/wp-calypso#50358 means I can't test on wp.com 😅 

I tried testing again on my standalone WP install, and for some reason I didn't get errors this time. Not sure what I was doing wrong last time, but no errors when I grabbed the latest theme folder from trunk.

Yep, this issue appears to be fixed in v2 of the theme! Thanks :)
